diag_test.py

```
@@ -398,31 +449,128 @@ def custom_sort(label):
+                elif lr_minus < 0.2: lr_minus_badge = get_badge_html("Moderate Rule-Out", "info")
+                elif lr_minus < 0.5: lr_minus_badge = get_badge_html("Weak Rule-Out", "warning")
+                else: lr_minus_badge = get_badge_html("No Value", "neutral")
+
+                # OR Badge
+                if or_value > 3 or or_value < 0.33: or_badge = get_badge_html("Strong Association", "success")
+                elif or_value > 2 or or_value < 0.5: or_badge = get_badge_html("Moderate Association", "info")
+                else: or_badge = get_badge_html("Weak/None", "neutral")
+                
+                # --- FORMAT STRINGS WITH HIGHLIGHTING ---
+                # RR CI
+                rr_ci_str = f"{rr_ci_lower:.4f}–{rr_ci_upper:.4f}" if np.isfinite(rr_ci_lower) else "NA"
```

Ruff (0.14.10)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)